back/app/models/nav_bar_item.rb

```diff
@@ -39,8 +39,8 @@ def custom?
     code == 'custom'
   end
 
-  def title_multiloc
-    super.presence || fallback_title_multiloc
+  def title_multiloc_with_fallback
```

Comment:
In the code before this PR, I do not find a test for the original `title_multiloc` method, nor for `fallback_title_multiloc`.
There is no test for `title_multiloc_with_fallback` method in this PR. Probably you intended to cover it with the acceptance test, but I think this method deserves its own test, because it is a crucial part of the problem that you are solving in this PR. The fact that it merges 2 multilocs is special and should be described.
I also think that `fallback_title_multiloc` should be tested, given that there are 2 conditions in there that are harder to test in a higher level test.

Reply:
`title_multiloc_with_fallback` is already tested here: https://github.com/CitizenLabDotCo/citizenlab/pull/1933/files#diff-ff5535694688ca824ee95fe8ee8b9320e2ecf9d09536964e411860d7b199c7deR10

I don't think it's necessary to test `title_multiloc` or `fallback_title_multiloc`. `title_multiloc` is now just implemented by Rails and is no longer customized. `fallback_title_multiloc` is a private method.

Reply:
I do not understand how I overlooked. Good that it exists. 😃